src/core/reducers/errors.js

```diff
@@ -69,7 +69,7 @@ function getMessagesFromError(error) {
   }
 
   if (!errorData.messages.length) {
-    log.warn('Error object did not contain any messages', error);
+    log.warn(`Error object did not contain any messages: ${error}`);
```

Comment:
I guess the intention of logging this was to see the full data structure of `error`, to see what `error.response.data` looks like (if it exists). But, I dunno, I guess it is fine how you have it. We can always revisit it if we need to debug a related problem.